### ISSUE TITLE
Twitterscraper not working

### ISSUE THREAD

**Issue by bimbaba** [taspinar/twitterscraper#343]
The twitterscraper library not returning any tweets fetched for query_tweets or user modules..

Please help!

**Comment by lydia1990** [comment]
Hello, it is the same thing for me help please

**Comment by magicaltoast** [comment]
Yeap, I tried this library in the past and it was working as expected, but now where the time has come to do something useful with it suddenly I stopped working. I have an issue like the people above, query_tweets don't return anything

**Comment by bimbaba** [comment]
You can use the GetOldTweets3 python library to fetch tweets as an alternative to Twitter scraper. It’s very fast, easy to implement and gives a more coordinated columns of a tweet object.

**Comment by Aivi001** [comment]
> Yeap, I tried this library in the past and it was working as expected, but now where the time has come to do something useful with it suddenly I stopped working. I have an issue like the people above, query_tweets don't return anything

Yes, I'm also facing the same issue, got the following error message after "getting 0 tweets":

ERROR: ConnectionError HTTPSConnectionPool(host='twitter.com', port=443): Max retries exceeded with url: /search?f=tweets&vertical=default&q=%23Harami%20since%3A2020-09-02%20until%3A2020-09-04&l=en (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5a41956710>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

**Comment by GivenToFlyCoder** [comment]
Facing the same issue

**Comment by LinqLover** [comment]
> GetOldTweets3

Does not work either, at least for me. Raises HTTP 404 Errors ...
See Mottl/GetOldTweets3#98.

**Comment by gautampal1947** [comment]
> The twitterscraper library not returning any tweets fetched for query_tweets or user modules..
> 
> Please help!

Same issue

**Comment by xohyun** [comment]
Same issue!! help!!

**Comment by giulionf** [comment]
Maybe this is helping? https://github.com/Foo-Manroot/tweet-feed/issues?q=is%3Aissue+is%3Aclosed

**Comment by Altimis** [comment]
All twitter scrapers dont seem to work anymore. I tried to use selenium to simply scrap maximum tweets between two chosen dates for given quiries. Check my work on : [Scweet](https://github.com/Altimis/Scweet). let me know if you need any clarification.

**Comment by Abelsihombing** [comment]
Same issue ! Please Help !!!

**Comment by iSamBa** [comment]
Facing the same issue !
